missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/game/LobbyListener.java

```diff
@@ -64,22 +64,11 @@ public void onInteract(PlayerInteractEvent event) {
             if (!player.hasPermission("mw.change")) return;
 
             String displayName = event.getItem().getItemMeta().getDisplayName();
-
-            // same team:
-            if (displayName.equals(getGame().getPlayer(player).getTeam().getFullname())) return;
-
-            // too late for team change:
-            if (getGame().getTaskManager().getTimer().getSeconds() < 10) {
-                player.sendMessage(Messages.getMessage(true, Messages.MessageEnum.TEAM_CHANGE_TEAM_NO_LONGER_NOW));
-                return;
-            }
```

Comment:
Moved to `UserCommands.java`. Then we have all the necessary checks in the same place and no differences between the GUI interaction and the manual execution of the command.